src/vast/actor/sink/csv.cc

```diff
@@ -54,7 +54,7 @@ struct csv_printer : printer<csv_printer> {
     }
 
     std::string operator()(std::string const& str) const {
-      return util::byte_escape(str, ",+- \"");
+      return "\""+util::ReplaceString(str,"\"","\"\"")+"\"";
```

Comment:
Interesting, didn't know that CSV uses dual-doublequotes as poor-man's escaping. Is escaping with backslash (`\"`) also common practice?